### ISSUE TITLE
Windows: Don't create multiple entries in Explorer

### ISSUE THREAD

**Issue by xobs** [haiwen/seadrive-gui#264]
Seadrive sometimes creates multiple entries in the sidebar, even for the same repository. It should not do this.

![image](https://user-images.githubusercontent.com/238325/98491480-ebefaf00-226f-11eb-80fe-1b05aeff6ab8.png)

**Comment by killing** [comment]
Please upgrade the latest version. This has been fixed.

**Comment by xobs** [comment]
Thanks. I thought I was on the latest version, because the software claimed there wasn't anything newer:

![image](https://user-images.githubusercontent.com/238325/99161391-d7f2f400-272c-11eb-97c3-4eac4dd86256.png)

I'll try manually installing the latest version.

**Comment by xobs** [comment]
I installed the latest version, and it didn't remove the old entries.

I ended up doing the following to remove the old entries:

1. Uninstall SeaDrive 2.0.8 and remove account information
2. Open regedit
3. Search for "seadrive"
4. Remove every CLSID entry that contains seadrive. There are several under `HKEY_CLASSES_ROOT\CLSID\` and several corresponding ones under `HKEY_CLASSES_ROOT\WOW6432Node\CLSID\`.
5. Reinstall seadrive

There are two issues here, which weren't fixed by updating to 2.0.8:

1. Seadrive shouldn't create multiple entries for one server
2. Seadrive should remove all entries when it is uninstalled
